### ISSUE TITLE
mouse scrolling doesn't work in the Information tab

### ISSUE THREAD

**Issue by DolphinStKom** [multani/sonata#66]
when unfolding lyrics, and it does not fit in the screen you can not use the mouse scroll
for example:
![sonatainfoex](https://cloud.githubusercontent.com/assets/4992884/3734732/3ce6d3a0-171b-11e4-8f2b-8032ff5d629b.jpg)

**Comment by multani** [comment]
Yes, I faced this today: scrolling with the mouse wheel when the cursor is over the lyrics frame doesn't scroll anything, and it certainly doesn't scroll the rest of the window.

**Comment by svxsvx** [comment]
System: FreeBSD 10.2-release
Pyhton: 2.7
Sonata: 1.6.21 (from ports)

I'm really happy that I can say that at least once in my life because usully I'm the one with the extraordinary bugs but: works fine for me guys.
